open_event/static/js/jquery/jquery.smartWizard.js

```diff
@@ -0,0 +1,489 @@
+/*
+ * SmartWizard 3.3.1 plugin
+ * jQuery Wizard control Plugin
+ * by Dipu
+ *
+ * Refactored and extended:
+ * https://github.com/mstratman/jQuery-Smart-Wizard
+ *
+ * Original URLs:
+ * http://www.techlaboratory.net
+ * http://tech-laboratory.blogspot.com
+ */
+
+function SmartWizard(target, options) {
+    this.target       = target;
+    this.options      = options;
+    this.curStepIdx   = options.selected;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 space characters but found 4.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2663115097/issues/source?bid=3233864&fileBranchId=3324174#l17)